rust-src/bulletproofs/src/range_proof.rs

```diff
@@ -57,53 +67,6 @@ fn two_n_vec<F: Field>(n: u8) -> Vec<F> {
     two_n
 }
 
-/// This function takes one argument n and returns the
-/// vector (z^j, z^{j+1}, ..., z^{j+n-1}) in F^n for any field F
-/// The arguments are
-/// - z - the field element z
-/// - first_power - the first power j
-/// - n - the integer n.
-fn z_vec<F: Field>(z: F, first_power: usize, n: usize) -> Vec<F> {
-    let mut z_n = Vec::with_capacity(n);
-    let mut z_i = F::one();
-    // FIXME: This should would be better to do with `pow`.
-    for _ in 0..first_power {
-        z_i.mul_assign(&z);
-    }
-    for _ in 0..n {
-        z_n.push(z_i);
-        z_i.mul_assign(&z);
-    }
-    z_n
-}
-
-/// Struct containing generators G and H needed for range proofs
-#[allow(non_snake_case)]
-#[derive(Debug, Clone, Serialize, SerdeBase16Serialize)]
-pub struct Generators<C: Curve> {
-    #[size_length = 4]
-    pub G_H: Vec<(C, C)>,
-}
```

Comment:
This library is exported as part of the rust SDK. Removing this type from it is a needless breaking API change.

Please re-export this type from the module.

Reply:
We need the struct also for the set-membership proof. We could import it there from the range-proof, but that seems the wrong place to have it. What is the proper way to resolve this?

Reply:
It is what I suggested, to re-export.

```
pub use utils::Generators;
```

Reply:
Done